tensorflow_addons/image/transform_ops.py

```diff
@@ -223,7 +225,7 @@ def angles_to_projective_transforms(angles,
         elif len(angle_or_angles.get_shape()) == 1:
             angles = angle_or_angles
         else:
-            raise TypeError("Angles should have rank 0 or 1.")
+            raise ValueError("Angles should have rank 0 or 1.")
```

Comment:
ditto
```suggestion
            raise ValueError("angles should have rank 0 or 1.")
```

Reply:
Do we prefer to use lower case for `angles`?